source/uninstall_objects.sql

```diff
@@ -137,10 +137,6 @@ drop type ut_data_value_boolean force;
 
 drop type ut_data_value_blob force;
 
-drop type ut_data_value_object force;
```

Comment:
Why those drops are removed?

Reply:
This are the types that been removed with introduction of ANYDATA into cursor. I can create separate issue on this .

Reply:
Cool